src/main/java/com/alpsbte/plotsystem/utils/Shortlink.java

```diff
@@ -0,0 +1,81 @@
+package com.alpsbte.plotsystem.utils;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+public class Shortlink {
+
+    public static String generateShortlink(String linkToShorten, String apikey, String host) throws IOException {
+        String fullURL = URLEncoder.encode(linkToShorten, StandardCharsets.UTF_8.toString());
+        URL url = new URL(host + "/rest/v2/short-urls/shorten?apiKey=" + apikey + "&format=txt&longUrl=" + fullURL);
+        HttpsURLConnection con = (HttpsURLConnection) url.openConnection();
+        con.setRequestMethod("GET");
+        con.setRequestProperty("User-Agent", "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.11 (KHTML, like Gecko) Chrome/23.0.1271.95 Safari/537.11");
+        con.addRequestProperty("Accept",
+                "*/*");
+        con.connect();
+        BufferedReader in = new BufferedReader(
+                new InputStreamReader(con.getInputStream()));
+        String inputLine;
+        StringBuffer content = new StringBuffer();
+        while ((inputLine = in.readLine()) != null) {
+            content.append(inputLine);
+        }
+        in.close();
+        return content.toString();
+    }
+
+    public static String generateShortlink(String linkToShorten, String apikey) throws IOException {
+        String fullURL = URLEncoder.encode(linkToShorten, StandardCharsets.UTF_8.toString());
+        URL url = new URL("https://buildthe.earth/rest/v2/short-urls/shorten?apiKey=" + apikey + "&format=txt&longUrl=" + fullURL);
```

Comment:
Unless this domain and api is run entirely by main bte, I would remove such a default option as it goes directly against our open and neutral goal for this project!